app-web/src/components/Cards/Card/Card.js

```diff
@@ -36,7 +36,7 @@ import { RESOURCE_TYPES_LIST } from '../../../constants/ui';
 
 const Card = ({ type, title, description, image, link, theme }) => {
   // console.log(theme, type);
-  let isExternal = validUrl.isWebUri(link);
+  let isExternal = !!validUrl.isWebUri(link);
```

Comment:
this was to address a proptype error, this function is turns out does not return a boolean when a link __is__ a web uri, it returns a string instead. using `!!` is an easy way to cast falsy/truthy values